src/Avalonia.Controls/Primitives/PopupPositioning/ManagedPopupPositionerPopupImplHelper.cs

```diff
@@ -27,12 +27,12 @@ public IReadOnlyList<ManagedPopupPositionerScreenInfo> Screens
         {
             get
             {
-                if (_parent.Screen is null)
+                if (_parent.TryGetFeature<IScreenImpl>() is not { } screenImpl)
```

Comment:
Why do we need to use a private API here? Is the public API no longer enough?

Reply:
Public API doesn't accept WindowsBaseImpl, it accents WindowsBase. So I would need to use private API either way.